src/js/ttsHookup.js

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.firstDiscovery");
+
+    fluid.defaults("gpii.firstDiscovery.tts.hookup", {
```

Comment:
Best to defer this problem - it can be handled once we have my FLUID-5249 branch in as well as cindyli's model relay upgrade - then we can rely on model relay to keep all such state synchronised across the entire component tree.